### PR TITLE
Add a first version of MetaShARK Interactive tool

### PR DESCRIPTION
**Pull request by yvanlebras**
Here is a first version of MetaShARK interactive tool GxIT. Here nothing is managed as input or output, need to work on it as there is no link between datasets from Galaxy or output.

### CODE REVIEW

**Comment by bgruening** [comment]
For the time beingz those ITs should go directly into the Galaxy main repository.

**Comment by yvanlebras** [comment]
For now a version with no direct interaction with Galaxy (no input from Galaxy history and no output to Galaxy history)

**Comment by yvanlebras** [comment]
What do you think @bgruening to propose to have such interactive tool where Galaxy can be seen "just" as a R Shiny apps ""scheduler"" ?

File: tools/interactive/interactivetool_metashark.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <command><![CDATA[
+
+        mkdir -p /srv/shiny-server/data/ &&
+        cp '$infile' /srv/shiny-server/data/inputdata.txt &&
+        R -e "options('shiny.port'=3838,shiny.host='0.0.0.0');MetaShARK::runMetashark()"
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="infile" type="data" format="tabular,csv" label="csv file"/>
```
Comment (bgruening) [inline tools/interactive/interactivetool_metashark.xml:21]: fix label please

File: tools/interactive/interactivetool_metashark.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<tool id="interactive_tool_metashark" tool_type="interactive" name="metashark" version="1.6">
```
Comment (bgruening) [inline tools/interactive/interactivetool_metashark.xml:1]: version number does not fit the container ? maybe as a macro token?

**Review by bgruening — COMMENTED** [review]
@yvanlebras this should be submitted to the main Galaxy repo. ITs should not be put into the TS (yet).

File: tools/interactive/interactivetool_metashark.xml (hunk 2 above)
Comment (yvanlebras) [reply tools/interactive/interactivetool_metashark.xml]: You're right! Thank you!

File: tools/interactive/interactivetool_metashark.xml (hunk 1 above)
Comment (yvanlebras) [reply tools/interactive/interactivetool_metashark.xml]: Thank you !

**Comment by yvanlebras** [comment]
Done!

**Pull request closed without merging**